tests/integration/instant_benchmark.py

```diff
@@ -219,23 +236,35 @@ def machine_translation(machine_name: str):
         return "lmi"
 
 
-def build_running_script(template, job, instance):
+def build_running_script(template, job, instance, record):
     with open(template) as f:
         template = json.load(f)
     job_template = template[job]
-    job_template['awscurl'] = bytes.fromhex(
-        job_template['awscurl']).decode("utf-8")
+    for key in job_template['awscurl'].keys():
+        job_template['awscurl'][key] = bytes.fromhex(
+            job_template['awscurl'][key]).decode("utf-8")
     write_model_artifacts(job_template['properties'],
                           job_template['requirements'], job_template['env'])
-
     container = job_template['container']
 
+    benchmark_command = ['set -x']
+    record_benchmark = ('python3 record_benchmark.py --template template.json '
+                        f'--job {job} --instance {instance} '
+                        f'--model models/test --record {record}')
+
+    for key, value in job_template['awscurl'].items():
+        benchmark_command.append("rm -rf benchmark_result.json benchmark.log")
+        benchmark_command.append(value)
+        benchmark_command.append(record_benchmark +
+                                 f' --benchmark-vars "{key}"')
+
     bash_command = [
-        'set -euo pipefail', 'echo "Start Launching container..."',
+        'set -euo pipefail',
+        'echo "Start Launching container..."',
         f"docker pull {container}",
         f"./launch_container.sh {container} $PWD/models {machine_translation(instance)}",
-        job_template['awscurl'] + " | tee benchmark.log"
     ]
+    bash_command.extend(benchmark_command)
```

Comment:
nit, but can we just use the same variable for benchmark_command and bash_command. Then all the parts run will be in the file in order